openeogeotrellis/geopysparkdatacube.py

```diff
@@ -312,6 +312,36 @@ def add_dimension(self, name: str, label: str, type: str = None):
             metadata=self.metadata.add_dimension(name=name, label=label, type=type)
         )
 
+    def drop_dimension(self, dimension: str = "other"):
+        if dimension not in ['bands', 'temporal']:
```

Comment:
as noted in https://github.com/Open-EO/openeo-python-client/pull/257  `drop_dimension` should drop based on dimension name, not dimension type

Reply:
and related, from https://processes.openeo.org/#drop_dimension : 
> If a dimension with the specified name does not exist, the process fails with a DimensionNotAvailable exception.

Reply:
also, it's probably better to use same parameter name `name` instead of `dimension`

see https://processes.openeo.org/#drop_dimension  and https://github.com/Open-EO/openeo-processes/issues/305